.github/workflows/publish.yml

```
@@ -1,39 +1,159 @@
 #      run: exit 1
+    # Sunrise edit end
+
+    # Sunrise added start - проверяем ветку, заданную пресетом
+    - name: Fail if not on required branch
+      if: ${{ inputs.required_branch != '' && github.ref_name != inputs.required_branch }}
+      run: exit 1
+    # Sunrise added end
 
+    # Sunrise edit start - устанавливаем зависимости публикации через Python
     - name: Set up Python
       uses: actions/setup-python@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)